### PR TITLE
feat: adds support for stdout.

### PR DESCRIPTION
**Pull request by jcchavezs**
This PR adds support for stdout as output for debug log. Most of the references to stdout in modsec world points to `/dev/stdout` as the target although it isn't idiomatic to Windows.

> Thank you for contributing to Coraza WAF, your effort is greatly appreciated
> Before submitting check if what you want to add to `coraza` list meets [quality standards](https://github.com/corazawaf/coraza/blob/master/CONTRIBUTING.md#quality-standards) before sending pull request. Thanks!

**Make sure that you've checked the boxes below before you submit PR:**

- [ ] My code includes positive and negative tests.
- [ ] I have an appropriate description with correct grammar.
- [ ] I have read [Contribution guidelines](https://github.com/corazawaf/coraza/blob/master/CONTRIBUTING.md#contribution-guidelines), [maintainers note](https://github.com/corazawaf/coraza/blob/master/CONTRIBUTING.md#maintainers) and [Quality standard](https://github.com/github.com/corazawaf/coraza/v3sso/coraza-waf/blob/master/CONTRIBUTING.md#quality-standards).
- [ ] My code is properly linted and passes pre-commit tests.

Thanks for your contribution :heart:

### CODE REVIEW

File: internal/corazawaf/waf.go [hunk 1]
```diff
@@ -420,9 +420,15 @@ func (w *WAF) SetDebugLogPath(path string) error {
 		w.Logger.SetOutput(io.Discard)
 		return nil
 	}
+
+	if path == "/dev/stdout" {
```
Comment (jcchavezs) [inline internal/corazawaf/waf.go:424]: This is the important change.

Comment (fzipi) [reply internal/corazawaf/waf.go:424]: What about using stderr? Should this deserve also a case for it?

**Comment by jcchavezs** [comment]
@fzipri actually this is debug log and most of the usages I saw (did a
quick research in GitHub/Google) was pointing to /dev/stdout. More so this
started as a need for us to display logs in stdout so I would hold the
addition of stderr until someone request for it.

File: internal/corazawaf/waf.go (hunk 1 above)
Comment (anuraaga) [inline internal/corazawaf/waf.go:424]: I understand a lot of current usage is for stdout, but I think it's too conventional to treat stdout / stderr as siblings to skip the latter here. Also IMO, stderr seems more appropriate for debug logs anyways

**Comment by jcchavezs** [comment]
I added support for `stderr` as per @fzipi and @anuraaga suggestion.

**Review by anuraaga — APPROVED** [review]

**Comment by fzipi** [comment]
Approved! :)

**Pull request merged**